src/snowcli/cli/snowpark/common.py

```diff
@@ -43,6 +43,15 @@ def print_log_lines(file: TextIO, name, id, logs):
         print(_prefix_line(prefix, log + "\n"), file=file, end="", flush=True)
 
 
+def remove_parameter_names(identifier: str):
```

Comment:
Can you add one-liner docstring explaining what and why we do?

Reply:
Added